UI/src/app/services/backend-connector.service.ts

```diff
@@ -72,6 +74,30 @@ export class BackendConnectorService {
         //     });
     };
 
+   fileUpload(fileItem:File, extraData?:object):any{
+      var apiCreateEndpoint = 'http://localhost:3001/api/upload';
```

Comment:
please use relative url, we have proxy in place.

Reply:
I changed to relative URL